src/objects/InstancedMesh.js

```diff
@@ -13,6 +13,8 @@ function InstancedMesh( geometry, material, count ) {
 
 	this.count = count;
 
+	this.type = 'InstancedMesh';
```

Comment:
Hmm, what's this for?

Reply:
I think we need this for serialization/deserialization. Adding support for `InstancedMesh` in `ObjectLoader` was my next task^^

Reply:
Wait! I've realized while working on this that `type` is not needed^^. Removing it here: 00e137184b23fe681eebbe263c879dee8aaecd01

Reply:
Thanks!